gemfiles/rails_4.gemfile.lock

```diff
@@ -0,0 +1,224 @@
+GIT
```

Comment:
the `.lock` files should be ignored from github

Reply:
done